lib/browser/api/power-monitor.ts

```diff
@@ -10,19 +10,35 @@ Object.setPrototypeOf(PowerMonitor.prototype, EventEmitter.prototype)
 
 const powerMonitor = createLazyInstance(createPowerMonitor, PowerMonitor, true)
 
-// On Linux we need to call blockShutdown() to subscribe to shutdown event.
 if (process.platform === 'linux') {
-  powerMonitor.on('newListener', (event:string) => {
-    if (event === 'shutdown' && powerMonitor.listenerCount('shutdown') === 0) {
-      powerMonitor.blockShutdown()
-    }
-  })
+  /* In order to delay system shutdown when e.preventDefault() is invoked
```

Comment:
While we don't have a guide on comments, most files are using `//` for multi-line comments, I think it is better being consistent here.

Reply:
Sounds good. Changed in ff0e17428827dbadbe2f97190a5483494f39192b